daprdocs/content/en/reference/arguments-annotations-overview.md

```diff
@@ -36,9 +36,11 @@ This table is meant to help users understand the equivalent options for running
 | `--profiling-port` | `--profiling-port` | | not supported | The port for the profile server (default "7777") |
 | `--app-protocol` | `--app-protocol` | `-P` | `dapr.io/app-protocol` | Tells Dapr which protocol your application is using. Valid options are `http` and `grpc`. Default is `http` |
 | `--sentry-address` | `--sentry-address` | | not supported | Address for the Sentry CA service |
+| `--sentry-address` | `--sentry-address` | | not supported | Address for the Sentry CA service |
```

Comment:
Looks like this is a duplicate row. Please remove

Reply:
Copy and paste error.